src/test/incremental/hashes/inherent_impls.rs

```diff
@@ -42,7 +42,10 @@ impl Foo {
 #[rustc_clean(cfg="cfail2")]
 #[rustc_clean(cfg="cfail3")]
 impl Foo {
-    #[rustc_clean(cfg="cfail2", except="HirBody,optimized_mir,mir_built,typeck_tables_of")]
+    #[rustc_clean(
+        cfg="cfail2",
+        except="HirBody,optimized_mir,mir_built,typeck_tables_of"
```

Comment:
hmmm, this should have contained `promoted_mir`, too, since that's nonempty when you do printing